src/BuildingRegistry/Building/Building.cs

```diff
@@ -56,6 +56,17 @@ public void ChangeOutlining(ExtendedWkbGeometry extendedWkbGeometry)
         {
             GuardRemovedBuilding();
 
+            var invalidStatuses = new List<BuildingStatus>
```

Comment:
should be initialized once (static readonly)

Reply:
Moved to `private static readonly List<BuildingStatus> ChangeBuildingOutlineInvalidStatuses` member